html/pages/device/nac.inc.php

```diff
@@ -69,9 +69,40 @@
         formatters: {
             "time_interval": function (column, row) {
                 var value = row[column.id];
-                return moment.duration(Number(value), 's').humanize();
+                var duration = moment.duration(Number(value), 's');
```

Comment:
This should probably just be a function in librenms.js

Reply:
OK. Will do.

Reply:
done